apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c

```diff
@@ -1041,6 +1056,7 @@ void tm_utc_main(void)
 
 	utc_taskmanager_dealloc_broadcast_msg_n();
 	utc_taskmanager_dealloc_broadcast_msg_p();
+
```

Comment:
Please remove unnecessary this empty line.

Reply:
modify